src/app/dim-ui/Dropdown.m.scss

```diff
@@ -52,6 +52,10 @@
 
 .disabled {
   color: #999 !important;
+  filter: grayscale(1);
+  opacity: 0.5;
+  cursor: not-allowed;
```

Comment:
I'll remove this along with the menu desaturation. 

This was an attempt to be consistent with the the not-allowed cursor on disabled buttons in the organisor page. I personally always found the dropdown menu in the search field to be confusing because the entire dropdown is essentially redundant until you enter a search term. The not-allowed cursor (along with the desaturation) makes it more unambiguous that the actions are disabled — however, there's also a case to be made that the vertical ellipsis button itself should only appear after a search term is entered… either way something that can be bundled in a PR dedicated to improving how disabled menus/actions appear.

Reply:
My idea behind showing the menu but having it greyed out is to help folks understand that there are actions you can take on a search - otherwise, until they search there's no way to know what you can do. I could buy an argument that it's even more enticing to see a button appear though...